### ISSUE TITLE
Microsoft Mouse and Keyboard Center [Microsoft.MouseandKeyboardCenter] doesn't set 3010 as successful return code

### ISSUE THREAD

**Issue by chbwien** [microsoft/winget-pkgs#23748]
# Brief description of your issue

 Microsoft Mouse and Keyboard Center [Microsoft.MouseandKeyboardCenter] sends the return code 3010, but obviously, this code was not set as a success code in the manifest
# Steps to reproduce

Install the packet and the 'error code' 3010 is returned, winget says installer failed

# Expected behavior

3010 should be set as success code

# Actual behavior

3010 is evaluated as failed.

# Environment

```none
[winget --info]
Windows Package Manager v1.0.11692
Windows: Windows.Desktop v10.0.19043.1110
Paket: Microsoft.DesktopAppInstaller v1.12.11692.0

Any other software?
```

I suggest adding this in the manifest
  InstallerSuccessCodes:
    - 3010

**Comment by ImJoakim** [comment]
> 3010 seems to be the installer failing (after a 3010 on my sandbox, the app isn't present). Can you post the output of `winget list` after an install of Mouse and Keyboard Center? (Also, did you reboot after the install?)

It doesn't seem to be able to install on a virtual machine / sandbox. When I tried installing it with interactive, this message popped up:

![image](https://user-images.githubusercontent.com/52826317/128245609-789a2c4b-b9f6-4e73-9cc1-e1d45abff93d.png)

When I click OK, it does actually say it's been successfully installed, but it obviously hasn't.


Also, when I try to install it on an actual machine, it installs just fine (ignore the Norwegian language :p):

![image](https://user-images.githubusercontent.com/52826317/128245768-556f7cf2-57c8-423c-acd7-4b7f090f4cd4.png)

**Comment by jedieaston** [comment]
I wonder how it got into the repo in the first place, I'd think it would fail validation due to it not installing in the VM.

**Comment by ImJoakim** [comment]
I'm guessing the `/silentinstall` installer switch suppresses the popup message regarding VM.

**Comment by denelon** [comment]
@jedieaston this is likely an older package from before automated VM validation. Some of the "early" packages were tested on standard desktop machines.

**Comment by chbwien** [comment]
Sorry for the late reply.
winget list tells me AFTER the upgrade:
Name                                                     ID                                                        Version
-----------------------------------------------------------------------------------------------------------------------------------
Microsoft Mouse and Keyboard Center                      Microsoft.MouseandKeyboardCenter                          13.250.137.0

**Comment by chbwien** [comment]
Of course, it will not work on a vm when there is no Microsoft Mouse or Keyboard detected. :-D
I have the wireless desktop 3050 and it works great. But with the 3010 (reboot needed) return code, that is accidentally misread as a failure.
